$OpenBSD$

# XXX: This patch is bullshit and needs to be properly fixed. This just gets me
# around the build error for now for testing.

Index: libstdc++-v3/src/c++11/cxx11-ios_failure.cc
--- libstdc++-v3/src/c++11/cxx11-ios_failure.cc.orig
+++ libstdc++-v3/src/c++11/cxx11-ios_failure.cc
@@ -168,10 +168,10 @@ _GLIBCXX_BEGIN_NAMESPACE_VERSION
   __throw_ios_failure(const char* __s __attribute__((unused)))
   { _GLIBCXX_THROW_OR_ABORT(__ios_failure(_(__s))); }
 
-  void
-  __throw_ios_failure(const char* str __attribute__((unused)),
-		      int err __attribute__((unused)))
-  { _GLIBCXX_THROW_OR_ABORT(__ios_failure(_(str), err)); }
+  /* void */
+  /* __throw_ios_failure(const char* str __attribute__((unused)), */
+		      /* int err __attribute__((unused))) */
+  /* { _GLIBCXX_THROW_OR_ABORT(__ios_failure(_(str), err)); } */
 
 _GLIBCXX_END_NAMESPACE_VERSION
 } // namespace
